addons/stock/utils/overview_graph.py

```diff
@@ -0,0 +1,48 @@
+import pytz
+from datetime import timedelta
+
+from odoo import fields
+
+
+def calculate_date_category(record, datetime):
```

Comment:
It's probably a better idea to do it on res_users in private since record is always a user

Reply:
Or move it back to the old place since it would be only used there.
And the new function could be in picking_type class since it's directly related

Reply:
I've moved both methods to stock picking and removed the utils file. I have kept a separate file for tests though, I expect the graph functionality to grow in the future :)